build/helper.py

```diff
@@ -200,15 +222,15 @@ def get_library_call_parameter_types_snippet(parameters_list):
             snippets.append(x['ctypes_type'])
     return ', '.join(snippets)
 
-def _get_output_param_return_snippet(output_parameter):
+def _get_output_param_return_snippet(output_parameter, parameters):
     '''Returns the snippet for returning a single output parameter from a Session method, i.e. "reading_ctype.value"'''
     assert output_parameter['direction'] == 'out', pp.pformat(output_parameter)
     if output_parameter['is_buffer']:
         if output_parameter['type'] == 'ViChar' or output_parameter['type'] == 'ViString':
             snippet = output_parameter['ctypes_variable_name'] + '.value.decode("ascii")'
         else:
-            # TODO(marcoskirsch): I don't like calling camelcase_to_snakecase here, it relies on contract that parameter name where the size is stored was created with that function.
-            snippet = '[' + output_parameter['ctypes_variable_name'] + '[i].value for i in range(' + camelcase_to_snakecase(output_parameter['size']) + ')]'
+            size_parameter = find_size_parameter(output_parameter, parameters)
```

Comment:
This seems like it would have the same issue as above regarding not finding a parameter when ['size']['value'] is a size, and not another parameter name